src/Plugin/CKEditorPlugin/DrupalEntity.php

```diff
@@ -25,6 +26,17 @@ class DrupalEntity extends EmbedCKEditorPluginBase {
   /**
    * {@inheritdoc}
    */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    $configuration['inline'] = $container->get('config.factory')
```

Comment:
Oh wow that's quite some limitation of CKEditor then...

So if we need to limit this per button / entity type, we would need to have different tags. We can NOT depend on classes / attributes in the DTD.  But using multiple tags is also a mismatch to the current filter approach - it would need extra configuration and changing it contains risk stale data.

How about considering an autogenerated (optional?) "drupal-entity-TYPE" tag that would allow specific DTDs? Yeah quite some change / added complexity in the filter..